packages/edit-widgets/src/blocks/legacy-widget/edit/widget-preview.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { addQueryArgs } from '@wordpress/url';
+import { Disabled, FocusableIframe } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+
+function WidgetPreview( { widgetAreaId, attributes, ...props } ) {
+	const DEFAULT_HEIGHT = 300;
+	const HEIGHT_MARGIN = 20;
+	const [ height, setHeight ] = useState( DEFAULT_HEIGHT );
+	const currentUrl = document.location.href;
+	const siteUrl = currentUrl.substr( 0, currentUrl.indexOf( 'wp-admin/' ) );
```

Comment:
Anyone aware of any a better way of getting site's URL?

Reply:
It is available in the REST API index at `wp-json`.

Reply:
Actually there is another place in navigation editor that could also use this correction - let's address both in a separate PR